src/python/grpcio/tests/qps/control.proto

```diff
@@ -0,0 +1,196 @@
+// Copyright 2015, Google Inc.
```

Comment:
we are trying very hard not to copy the .proto files around (we've had enough problems with having multiple copies of the same .proto files), so the protos in src/proto should be used.